source/_patterns/02-molecules/hero/hero.twig

```diff
@@ -61,9 +61,16 @@
             {% endfor %}
           </div>
         {% endif %}
+
```

Comment:
No need to modify hero.twig.  You can pass any additional content to hero.main.lead.

In your template, set the variable like this:

```
{% set additional_content %}
    <div class="whatever-structure">
        ...
    </div>
{% endset %}
```

Then pass it to the hero like so in your template:

```
 {% include "@molecules/hero/hero.twig" with {
        hero: {
          layout: 'default',
          background: 'default',
          main: {
            title: label,
            lead: additional_content
          },
          aside: {
            text: download,
          },
          classes: []
        }
      } %}
```

The hero is generic enough to accept any structure.  There's no need to make it specialized for one use case.

Reply:
Implemented the code review suggestion.